src/loaders/med4_load.c

```diff
@@ -799,7 +799,9 @@ static int med4_load(struct module_data *m, HIO_HANDLE *f, const int start)
 		smp_idx++;
 	}
 
-	hio_read16b(f);	/* unknown */
+	/* Not sure what this was supposed to be, but it isn't present in
+	 * Synth-a-sysmic.med or any other MED4 module on ModLand. */
+	/*hio_read16b(f);*/	/* unknown */
```

Comment:
How did this not break iff size below, before? Did we, by luck, always hit the `goto parse_iff;` line or something?

Reply:
What I think was going on here:

* `parse_iff:` etc. doesn't have an error condition, it just attempts to read until EOF and then exits successfully (fair IMO).
* It would read the lower two bytes of the tag as the upper two bytes of the tag, and the upper two bytes of the size as the lower two bytes of the tag, meaning none of the IFF tag handlers would execute.
* It would read the lower two bytes of the size as the upper two bytes of the size, causing it to immediately seek past the end, EOF on reading the next tag, and break.